src/routes/mods/[slug]/+page.svelte

```diff
@@ -0,0 +1,194 @@
+<script>
+	import Tag from '$lib/components/text/Tag.svelte';
+    import Check from '$lib/icon/Check.svelte';
+    import SvelteMarkdown from 'svelte-markdown';
+
+    import FullLoader from '$lib/components/FullLoader.svelte';
+    import Link from '$lib/components/text/Link.svelte';
+
+	import { goto } from '$app/navigation';
+	import { page } from '$app/stores';
+	let slug = $page.params.slug;
+
+    import { selectedVersion } from '$lib/stores';
+    import { getPackageInfo, isInstalledForVersion, getDeplist } from '$lib/api/contentdb';
+    import { downloadAndUnzip } from '$lib/api/download';
+    //import { openGame } from '$lib/shell';
+
+    import { onMount } from 'svelte';
+  	import { writable } from 'svelte/store';
+	
+    let packageInfo = false;
+    let isInstalled = false;
+    let isSupported = false; //TODO: figure out a way to do this...
+
+	let installing = false;
+	let currentinstalling = writable("");
+	let progress = writable(0);
+	let modstoinstall = [];
+	
+    let [ author, pack ] = slug.split('@');
+
+    onMount(updatePackage);
+	page.subscribe(updatePackage);
+
+	async function updatePackage() {
+		slug = $page.params.slug;
+		let slugParts = slug.split('@');
+        packageInfo = await getPackageInfo(slugParts[0], slugParts[1]);
+		//isInstalled = await isInstalledForVersion(slugParts[1], 'mods', $selectedVersion.name);
+	};
+
+    selectedVersion.subscribe(async(val) => {
+        if (!val.installed) {
+			isSupported = false;
+			return;
+		}
+
+        isInstalled = await isInstalledForVersion(pack, 'mods', val.name);
+		for (let i in packageInfo.releasesdata){
```

Comment:
refactor in to a `for const of` loop

Reply:
`Uncaught (in promise) TypeError: packageInfo.releasesdata is not iterable` so we cannot